control/trajectory_follower_nodes/src/lateral_controller_node.cpp

```diff
@@ -266,6 +265,7 @@ void LateralController::onTrajectory(
   m_current_trajectory_ptr = msg;
 
   if (!m_current_pose_ptr && !updateCurrentPose()) {
+    RCLCPP_DEBUG(get_logger(), "Current pose is invalid!!");
```

Comment:
It is not invalid, just not received yet?

```suggestion
    RCLCPP_DEBUG(get_logger(), "Current pose is not received yet.");
```

Reply:
Thanks. It's better.